changelog/fix16002.dd

```diff
@@ -0,0 +1,4 @@
+fix Issue 16002 - Add __traits(isModule) and __traits(isPackage)
+
+This enhancement adds two new __traits, isModule and isPackage, which return
```

Comment:
You could surround the __traits, isModule, and isPackage in tick marks to make them stand out as D syntax in the rendered HTML pages.  For example:  "two new \`__traits\`, \`isModule\` and \`isPackage\`, which return ..."

Reply:
Same for the title "... Add \`__traits(isModule)\` and \`__traits(isPackage)\`"